contracts/GIVpower.sol

```diff
@@ -66,6 +68,9 @@ contract GIVpower is GardenUnipoolTokenDistributor, IERC20MetadataUpgradeable {
         if (rounds < 1) {
             revert ZeroLockRound();
         }
+        if (amount == 0) {
```

Comment:
Is there some reason to use `== 0` on this condition and `< 1` on the one above?

Reply:
No :), which one is preffered?

Reply:
Not a preference, I was wondering.